docs/wes-qc-hail.md

```diff
@@ -145,19 +207,19 @@ Next an input table is generated to run the random forest on.
 spark-submit 3-variant_qc/2-create_rf_ht.py
 ```
 
-Next train the random forest model, note the use of --master local[*] here which is needed to ensure that the worker nodes pick up the correct python modules.
+Next, train the random forest model, note the use of --master local[*] here which is needed to ensure that the worker nodes pick up the correct python modules.
```

Comment:
where is the --master local[*] arg?

Reply:
In the latest Hail, the RF model training works correctly in a parallel environment. I removed this sentence and placed it in the note below.